### ISSUE TITLE
Werte scheinen merkwürdig 

### ISSUE THREAD

**Issue by chrischmx** [skipperro/enpal-homeassistant#11]
Hallo zusammen,
erstmal, super Plugin! Allerdings nutze ich die neuste Version erst seit heute und die Werte sind mir etwas suspekt. 

![IMG_1286](https://github.com/user-attachments/assets/7cc6628d-3890-4886-ae18-465450917d4f)
![IMG_1285](https://github.com/user-attachments/assets/91201b8d-7835-40cb-9131-1981527c78db)
![IMG_1284](https://github.com/user-attachments/assets/11f01a3b-9f56-4952-84ee-f0ac892f83f8)
![IMG_1288](https://github.com/user-attachments/assets/2f054d93-0b40-4ae1-9d38-d4e9fb0bca63)

Ich weiß das nur die Werte aus der Influx DB verwendet werden. Aber hat wer eine Idee wie er drauf kommt das ich heute 54kw zu gekauft hätte? Also die Werte der selben Entität wird unterschiedlich dargestellt im Energie Dashboard als auch als Lächeln.

**Comment by skipperro** [comment]
Naja, wenn Update erst heute durchgefügt war, dann Energy Tab wird differenz zwischen letzte und aktuelle Wert als heutige verbrauch zeigen. So rechnet das HA. Ab morgen soll alles wieder richtig sein. Wenn nicht, sag bitte Bescheid.

**Comment by chrischmx** [comment]
Ah okay, danke das wusste ich nicht. Habe heute das Plugin installiert und gesehen das ich 15 Entitäten mehr habe und nun auch den Energie Tab befüllen kann.  Ich melde mich, dankeschön und einen Guten Rutsch ins neue Jahr

**Comment by chrischmx** [comment]
![IMG_1295](https://github.com/user-attachments/assets/3a32359e-ec54-4cd6-9491-adfefe9bd5af)
![IMG_1296](https://github.com/user-attachments/assets/1f3b87ec-e6c1-45a3-a1b5-2d361a0d86dd)

Guten Morgen, das sind nun die heutigen Werte

**Comment by skipperro** [comment]
-27 kWH auf den Solarzellen? Es sieht so aus, als hätten Sie die Registerkarte Energie in Ihrem Home Assistant falsch konfiguriert. Stellen Sie sicher, dass Sie dort nur Energiesensoren eingeben, die nur inkrementell sind, wie z. B. die insgesamt verbrauchte Energie, nicht die Energie pro Tag, die um Mitternacht auf 0 fällt, oder die aktuelle Leistung. 

Für den Anfang empfehle ich Ihnen, dies zu lesen:
https://www.home-assistant.io/docs/energy/electricity-grid/

**Comment by chrischmx** [comment]
Naja man sieht in den Quellen was ich angegeben habe. Andere Auswahlmöglichkeiten gibt's auch gar nicht.

**Comment by skipperro** [comment]
Ja, denn leider hat Enpal diese Daten nicht in seiner Datenbank, aber in HA kannst du Helfer für dich erstellen, die wie vorgesehen für den Energietab funktionieren.

Hier sind 3 Helfer, die ich für mich auf der Grundlage von Enpal-Daten definiert habe:
![image](https://github.com/user-attachments/assets/329c338e-0981-42ec-a650-254efeb5882c)
Und hier ist die Konfiguration der Registerkarte Energie:
![image](https://github.com/user-attachments/assets/4088529e-b733-4c38-a81a-d33760e71653)

Für Strommesser Werte (1.8.0 und 2.8.0) habe ich diese kleine Slimelezer benutzt:
https://www.zuidwijk.com/product/slimmelezer/

**Comment by chrischmx** [comment]
Ah jetzt kommen wir dem näher. Dachte ich kann die Werte so wie sie kommen verwenden. Muss mal sehen ob der Sleezer kompatibel mit unserem Zähler ist. Danke für den Hinweis, jetzt hab ich das Problem verstanden.

**Comment by skipperro** [comment]
Ich habe Zähler von Enpal - ISKRA AM550, und es funktioniert fast perfekt. Nur manchmal verliert es die WLAN-Verbindung und ich muss es ausstecken und wieder einstecken, um neu zu starten und die Verbindung wieder herzustellen. Ansonsten sollte es mit jedem Messgerät mit P1-Anschluss gut funktionieren.

**Comment by chrischmx** [comment]
Wir haben den Sagecom Smart4 der hat nur LMN Ports die man dann nutzen könnte für einen Smartmeter Gateway aber da muss wohl ein Pairing Key vom Hersteller geben den Enpal mit Sicherheit nicht rausrücken wird. Schade. Aber eig müsste es ja auch ohne gehen da ich ja die Zählerklemmen habe und in Enpal steht ja auch der tagesverbrauch, den könnte ich ja evtl auch mit einem Helper konsolidieren. Halt nicht so 100% korrekt aber immerhin etwas

**Comment by chrischmx** [comment]
Also, ich habe nun einen InfrarotLeser gekauft und der liefert mir nun zuverlässig den Gesamtstrom vom Zähler.
Das mit der Batterie etc klappt aber noch nicht so. Wäre es möglich das die Integration/Plugin zusätzlich Entitäten/Helfer erstellen kann für den Energie Tab? 
Hab mir mal diese Helfer hier erstellt und hoffe das klappt. Hatte bisher noch nichts damit zutun.

![Bildschirmfoto 2025-01-04 um 16 47 19](https://github.com/user-attachments/assets/c45a2ee2-ea4f-4bdc-aeee-af5b836efaa3)
